smart-alluxio/pom.xml

```diff
@@ -86,7 +93,16 @@
             <scope>test</scope>
             <type>test-jar</type>
         </dependency>
-
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
```

Comment:
In the main project guava is introduced but the version is 11.0.2 which is not compatible with required version 14.0.1. I am not sure that we can just modify {guava.version} in the main pom.xml.
Also I saw that we specify the guava version in other module such as smart-zeppelin.